pyvista/plotting/plotting.py

```diff
@@ -2166,7 +2173,31 @@ def add_scalar_bar(self, title=None, n_labels=5, italic=False,
 
         # Create scalar bar
         self.scalar_bar = vtk.vtkScalarBarActor()
-        self.scalar_bar.SetLookupTable(mapper.lookup_table)
+        if background_color is not None:
+            from ..core.common import vtk_to_numpy, numpy_to_vtk
+            if not isinstance(background_color, collections.Iterable):
```

Comment:
We should leverage `parse_color` here incase the user passes a string name of a color to be consistent with the rest of the API